### PR TITLE
Add phcue-ck recipe

### PR DESCRIPTION
**Pull request by andersgs**
Adds a new tools called phcue-ck that gets FTP links to FASTQ from ENA.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by andersgs** [comment]
@BiocondaBot please add label

File: recipes/phcue-ck/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - pkg-config
+    - make
+    - perl
+    - openssl
+  host:
+    - ca-certificates
+  run:
+   - ca-certificates
+    
+
```
Comment (pvanheus) [inline recipes/phcue-ck/meta.yaml:28]: Remove blank line.

File: recipes/phcue-ck/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  number: 0
+
+source:
+  url: https://github.com/lgi-onehealth/phcue-ck/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 067a79ac40aa7a1697f5dd7f09115e7d862eddaf975f4555fc67732e303e0d26
+
+requirements:
+  build:
+    - {{ compiler("c") }}
```
[the remark this answers is not in the document]
Comment (andersgs) [reply recipes/phcue-ck/meta.yaml]: I am attempting with nothing but rust and openssl as build dependencies.

File: recipes/phcue-ck/build.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -euo
```
Comment (marcelm) [inline recipes/phcue-ck/build.sh:1]: This looks weird. `-o` is normally followed by something to set, such as `-o pipefail`.

Comment (andersgs) [reply recipes/phcue-ck/build.sh]: Yes... Thank you! I had copy pasted from another rust recipe and did not notice it!

File: recipes/phcue-ck/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (andersgs) [reply recipes/phcue-ck/meta.yaml:16]: Ok. I seem to have found the minimum build requirements to get things working.

**Review by pvanheus — APPROVED** [review]

**Comment by pvanheus** [comment]
I'm still not sure why you need `make` when you're not using the Makefile in your build but I'm happy to approve this recipe in its current state. You can explain the `make` thing in the comments.

**Comment by andersgs** [comment]
Thank you @pvanheus. openssl-sys (I believe) crate is using it to compile openssl. Without it, the build fails.

**Comment by andersgs** [comment]
The docs here explain: https://docs.rs/openssl/latest/openssl/#vendored -- I am using the "vendored" feature to get statically linked binaries with musl. That is what I provide on the releases page on the repo.

**Comment by andersgs** [comment]
@BiocondaBot please merge

**Pull request merged**